Mathlib/Data/Real/Pi/Irrational.lean

```diff
@@ -0,0 +1,248 @@
+/-
+Copyright (c) 2022 Bhavik Mehta. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Bhavik Mehta
+-/
+import Mathlib.Analysis.SpecialFunctions.Integrals
+import Mathlib.Data.Real.Irrational
+
+/-!
+# `Real.pi` is irrational
+
+The main result of this file is `irrational_pi`.
```

Comment:
Can this docstring explain the main proof ideas?

Reply:
I added some more detail to the docstring.